### PR TITLE
update svi i example

### PR DESCRIPTION
**Pull request by martinjankowiak**
specifically, update svi i example to reflect the fact that it's using a repameterized beta distribution (n_samples -> 1).

### CODE REVIEW

File: tutorial/source/svi_part_iii.ipynb [hunk 1]
```diff
@@ -282,52 +323,51 @@
...
+    "                             constraint=constraints.positive)\n",
+    "        beta_q = pyro.param(\"beta_q\", torch.tensor(self.beta_q_0),\n",
+    "                            constraint=constraints.positive)\n",
     "        # sample f from the beta variational distribution\n",
     "        baseline_dict = {'use_decaying_avg_baseline': use_decaying_avg_baseline,\n",
     "                         'baseline_beta': 0.90}\n",
     "        # note that the baseline_dict specifies whether we're using\n",
     "        # decaying average baselines or not\n",
-    "        pyro.sample(\"latent_fairness\", dist.Beta(alpha_q, beta_q),\n",
+    "        f=pyro.sample(\"latent_fairness\", NonreparameterizedBeta(alpha_q, beta_q),\n",
```
Comment (fritzo) [inline tutorial/source/svi_part_iii.ipynb]: Did you add `f=` for debugging purposes? It does not seem to be used anywhere.

Comment (martinjankowiak) [reply tutorial/source/svi_part_iii.ipynb]: good catch, fixed

Comment (fritzo) [reply tutorial/source/svi_part_iii.ipynb]: push?

**Comment by fritzo** [comment]
bump

**Comment by martinjankowiak** [comment]
@fritzo ok resolved merge conflicts, sorry for the delay

**Pull request merged**